test/CMakeLists.txt

```diff
@@ -262,6 +262,7 @@ if(REALM_ENABLE_SYNC)
 	    test_sync.cpp
 	    test_sync_auth.cpp
 	    test_sync_history_migration.cpp
+            test_sync_subscriptions.cpp
```

Comment:
indentation inconsistency

Reply:
Ugh. Tabs. My old nemesis.

Reply:
Not sure how we ended up here, but the whitespace appears to be both a tab as well as 4 spaces 😞